apps/projects/contracts/Projects.sol

```diff
@@ -428,24 +469,28 @@ contract Projects is AragonApp {
     ) public isInitialized payable auth(ADD_BOUNTY_ROLE)
     {
         // ensure the transvalue passed equals transaction value
-        checkTransValueEqualsMessageValue(msg.value, _bountySizes,_tokenBounties);
-        
+        //checkTransValueEqualsMessageValue(msg.value, _bountySizes,_tokenBounties);
         string memory ipfsHash;
         uint standardBountyId;
         // submit the bounty to the StandardBounties contract
         for (uint i = 0; i < _bountySizes.length; i++) {
-            ipfsHash = substring(_ipfsAddresses, i.mul(46), i.add(1).mul(46));
+            ipfsHash = "";//substring(_ipfsAddresses, i.mul(46), i.add(1).mul(46));
```

Comment:
the hash should be passed in now that the integration is complete, right?

Reply:
Yeah, the substring function crashes it since it's not in constant time. I'll fix the substring function and then re-enable this.